<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminTableWindow</class>
 <widget class="QDialog" name="AdminTableWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>650</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <column>
    <property name="text">
     <string>Player_Name</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Birthday_Date</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Game_Played</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Goal</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Assist</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Point</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Penalty_Minutes</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="add_player_button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Add Player</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update_table_button">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>390</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Update Table</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_player_button">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>390</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Delete Player</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>691</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(16, 30, 41, 240);
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="logout_button">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>390</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Log Out</string>
   </property>
  </widget>
  <widget class="QPushButton" name="register_button">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>161</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Create a new account</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_account_window_button">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>161</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Delete existing account</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>tableWidget</zorder>
  <zorder>add_player_button</zorder>
  <zorder>update_table_button</zorder>
  <zorder>delete_player_button</zorder>
  <zorder>logout_button</zorder>
  <zorder>register_button</zorder>
  <zorder>delete_account_window_button</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
